components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -148,7 +193,11 @@ define([
 
             console.log("Export data: >>" + exportString + "<<");
             if (exportString != "") {
-                FileSaver.saveTextAs(exportString, fileName);
+                if (exportType != "image") {
+                    FileSaver.saveTextAs(exportString, fileName);
+                } else if (exportType == "image" && imageType == "svg") {
+                    FileSaver.saveTextAs(exportString, fileName, 'base64', 'data:image/png;base64');
```

Comment:
Disagree. "saveTextAs" already takes a charset as a third parameter.  If we need a separate function to set it to base64  why this parameter is here in a first place?
Having file type parameter there as an option seems pretty logical and clear. 
Very often file constructors take a file type as a standard parameter, like 
`var file = new File(["Hello, world!"], "hello world.txt", {type: "text/plain;charset=utf-8"});`
Why not having a file saving function with an option to specify type?